modules/rubiconAnalyticsAdapter.js

```diff
@@ -167,9 +167,10 @@ function sendMessage(auctionId, bidWonId) {
     referrerHostname: rubiconAdapter.referrerHostname || getHostNameFromReferer(referrer),
     channel: 'web',
   };
-  if (rubiConf.wrapperName || rubiConf.rule_name) {
```

Comment:
wrapperName is there always when rule_name is so we will only check it. same with new field family